app/models/user.rb

```diff
@@ -74,6 +74,10 @@ def confirmed?
     email_addresses.where.not(confirmed_at: nil).any?
   end
 
+  def confirmed_phone_configurations
```

Comment:
I see the field and code related to checking `confirmed_at`, but I'm not sure it's actually possible for someone to have a phone configuration record without it being confirmed. I just tested quickly in local development and the phone configuration isn't created until the code is entered (i.e. confirmed).

Which is all to say, maybe it's not necessary for us to check?

(And maybe a separate task to investigate if we still need `confirmation_sent_at` / `confirmed_at`, maybe they're related to legacy behaviors)

Reply:
alright we might not need to check. And I think another ticket to remove a column that isnt being used is a good idea.